### PR TITLE
Renaming sa_collectd -> collectd

### PR DESCRIPTION
**Pull request by csibbitt**
* See https://github.com/infrawatch/smart-gateway/pull/74#pullrequestreview-361954339

### CODE REVIEW

**Comment by csibbitt** [comment]
# UNTESTED
I would like to test a build of the smart gateway alongside this branch but probably won't end up doing it today. CI tests are going to fail unless the corresponding smart-gateway changes are published.

**Comment by csibbitt** [comment]
[Changes to smoketest/perftest/docs to remove "sa_" from the metrics prefix](https://trello.com/c/WKDRSOJX/928-changes-to-smoketest-perftest-docs-to-remove-sa-from-the-metrics-prefix)

**Comment by pleimer** [comment]
Sorry about all of the merged commits. But at least the CI bot address is fixed

File: ci.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 global:
         timeout: 2400
 script:
-        - oc login -u kubeadmin -p mhk2X-Y8ozE-9icYb-uLCdV --insecure-skip-tls-verify=true https://api.crc.testing:6443
+        - oc login -u kubeadmin -p sXrUX-mnUP4-oM4Lw-VbWUs --insecure-skip-tls-verify=true https://api.ciserver.lab.upshift.rdu2.redhat.com:6443
```
Comment (csibbitt) [inline ci.yml]: @pleimer Not sure why you committed these to my PR (this PR is never even expected to pass) but I'll assume you know what you're doing and aren't worried about merge conflicts.

**Comment by csibbitt** [comment]
# Testing

I deployed from this branch and got the expected `*** [FAILURE] Smoke test job still not succeeded after 300s`

Then I built the smart-gateway from https://github.com/infrawatch/smart-gateway/pull/74

```
[csibbitt@laptop smart-gateway ((a08eb57...))]$ docker build -t "${REGISTRY}/service-telemetry/smart-gateway" .
[...]
[csibbitt@laptop smart-gateway ((a08eb57...))]$ docker push "${REGISTRY}/service-telemetry/smart-gateway"
[...]
latest: digest: sha256:e2efd2de78bc1a17ccac69f7522c1e2eda61120a6376114e9a25c6ad3de2f12f size: 952
```

Deployed it into my environment:

```
$ oc describe po stf-default-telemetry-smartgateway-2-fw4vs | grep -i "Image ID"
    Image ID:      image-registry.openshift-image-registry.svc:5000/service-telemetry/smart-gateway@sha256:e2efd2de78bc1a17ccac69f7522c1e2eda61120a6376114e9a25c6ad3de2f12f
```

Re-ran smoketest:

```
$ ./tests/smoketest/smoketest.sh 
[...]
*** [SUCCESS] Smoke test job completed successfully
```

Then I ran a perftest and confirmed I could see stuff on the dashboard

```
$ ./performance-test.sh -t tb -c 60 -h 1 -p 1000 -i 1 -n 1
[...]
*** Test complete ***
```
Screenshot from 2020-02-25 19-10-04

**Comment by csibbitt** [comment]
Must merge with https://github.com/infrawatch/smart-gateway/pull/74

**Pull request merged**